Aagam_Shah/postgreAssignment/src/controller/school.controller.ts

```diff
@@ -0,0 +1,359 @@
+const db = require('../config/database').pool;
+import { Context } from "vm";
+
+function page_validation(page: number, size: number): {status: number, message: string}{
+  if (typeof page !== "number" || typeof size !== "number") {
+      return {status: 400,  message: "Invalid parameters"};
+  }
+  if(page < 0 || size <= 0){
+      return {status: 400, message: "Invalid parameters"};
+  }
+  return {status: 200, message: "true"};
+}
+
+function isValidId(uuid: string){
```

Comment:
no need to check if it valid uuid, if its not there in db then let throw error, while creating server will decide what will be the uuid, client shouldnt send uuid.

Reply:
understood